TodoApp.ConsoleApp/Repositories/XMLRepository/TodoRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Xml.Linq;
+using TodoApp.ConsoleApp.Repositories.Models;
+using TodoApp.ConsoleApp.Repositories.Interfaces;
+
+namespace TodoApp.ConsoleApp.Repositories.XMLRepository
+{
+    internal class TodoRepository : RepositoryBase<TodoModel, Guid>, ITodoRepository
```

Comment:
Add constructor for this class, which calls the base constructor.
- `path` is the path to the XML document. It should be provided from outside of this module (e.g. from configuration option; user input; fake path for testing, etc.)
- `"todos"` is the XML element name which holds the list of todos. We are hard-coding it, because we expect a specific XML format.

```cs
public TodoRepository(string path)
    : base(path, "todos")
{ }
```